### ISSUE TITLE
Cannnot run example code in Readme

### ISSUE THREAD

**Issue by Jacomichi** [Jij-Inc/Qamomile#75]
# Description

The code in Readme below
```python
from qamomile.core.qaoa import QAOAConverter
```
provides the error
```bash
ModuleNotFoundError: No module named 'qamomile.core.qaoa'
```

# How to fix
The code should be
```python
from qamomile.core.converters.qaoa import QAOAConverter
```

**Comment by Jacomichi** [comment]
I also found 
```python
import jijmodeling_transpiler as jmt
```
provides error.

It should be 
```python
import jijmodeling_transpiler.core as jmt
```

**Comment by Jacomichi** [comment]
This is fixed by #84
